### ISSUE TITLE
name 'unicode' is not defined

### ISSUE THREAD

**Issue by KottV** [BespokeSynth/BespokeSynth#19]
Hi,

Recent build throws this exception when loading any script:
```
37801: syntax highlight execution exception: NameError: name 'unicode' is not defined

At:
  <string>(18): syntax_highlight_basic
  <string>(2): <module>
```

**Comment by awwbees** [comment]
this is likely an issue related to differing versions of python installed. I'll have to do better error checking, but in the meantime, adding `return;` to the top of `CodeEntry::UpdateSyntaxHighlightMapping()` should take care of it (though you won't get any syntax highlighting)

**Comment by KottV** [comment]
Thanks, it helps.

**Comment by awwbees** [comment]
I fixed this issue in ac0255846327736e65dd1f556c97d5c0dc8d2e00, so now syntax highlighting works properly in both python2 and python3
